### ISSUE TITLE
Array Model Serialization in Safari Returns Object

### ISSUE THREAD

**Issue by danielrasmuson** [sylvainpolletvillard/ObjectModel#30]
```html
<script type="text/javascript" src="model.js"></script>
<script>
var list = Model.Array(String);
var result = JSON.stringify(new list(['one', 'two']));
console.log(result);

// chrome
// ["one","two"]

// safari 
// "{\"0\":\"one\",\"1\":\"two\"}"

</script>
```

I would expect safari to serialize an ObjectModel the same way as Chrome.

**Comment by sylvainpolletvillard** [comment]
What version of Safari are you using please ?

**Comment by sylvainpolletvillard** [comment]
Reproduced on Safari 5.1. I assume this is a bug from Safari since there has been [some recent bug fixes on JSON.stringify and arrays](https://trac.webkit.org/changeset/201674/trunk/Source). Anyway, I found a way to prevent the issue. Fixed in v2.3.2. Thanks for the report 👍

**Comment by danielrasmuson** [comment]
Thanks for the quick work Sylvain.
